package.json

```diff
@@ -87,27 +85,26 @@
     "pushover-notifications": "^1.2.0",
     "random-token": "0.0.8",
     "request": "^2.88.0",
+    "semver": "^5.6.0",
     "share2nightscout-bridge": "git://github.com/nightscout/share2nightscout-bridge.git#wip/generalize",
     "shiro-trie": "^0.4.8",
     "simple-statistics": "^0.7.0",
-    "socket.io": "^2.1.1",
-    "swagger-ui-dist": "^3.20.1",
+    "socket.io": "~2.1.1",
+    "swagger-ui-dist": "^3.20.4",
     "traverse": "^0.6.6",
-    "uglify-js": "^3.4.9",
-    "uuid": "^3.2.1"
+    "uglify-js": "^3.4.9"
   },
   "devDependencies": {
     "benv": "^3.3.0",
```

Comment:
I had to add `"jsdom": "~11.11.0",`.

`benv` brakes with 11.12 or later versions of jsdom.

Reply:
@jpcunningh great find. I didn't understand why my tests did not run under Windows, but the Travis builds were fine.  What platform do you use? I think we should try to seek for an alternative for `benv` because is does not seem to be maintained.

Reply:
@PieterGit, I'm running Ubuntu 18.04.